src/Microsoft.AspNetCore.Dispatcher/DispatcherMiddlewareExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Builder;
+
+namespace Microsoft.AspNetCore.Dispatcher
```

Comment:
This should be in the `Microsoft.AspNetCore.Builder` namespace. We do this so you don't need to a million namespaces to configure middleware.